packages/components/psammead-useful-links/src/index.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import styled from 'styled-components';
+import { getSerifMedium } from '@bbc/psammead-styles/font-styles';
+import { C_EBON, C_METAL } from '@bbc/psammead-styles/colours';
+import { grid } from '@bbc/psammead-styles/detection';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { string, shape, node } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import {
+  GEL_SPACING, // 8 px
+} from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+
+export const UsefulLinkItem = styled.a`
+  ${({ script }) => script && getPica(script)};
+  ${({ service }) => getSerifMedium(service)};
+  color: ${C_EBON};
+  text-decoration: none;
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+
+  &:visited {
+    color: ${C_METAL};
+  }
+`;
+
+const StyledUsefulLinksUl = styled.ul`
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    @supports (${grid}) {
+      display: grid;
+      grid-template-columns: auto auto;
+    }
+  }
+`;
+
+export const UsefulLinksUl = ({ children, ...props }) => (
+  <StyledUsefulLinksUl role="list" {...props}>
+    {children}
+  </StyledUsefulLinksUl>
+);
+
+const StyledUsefulLinksLi = styled.li`
+  padding-top: ${GEL_SPACING};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    @supports not (${grid}) {
+      display: inline-block;
+      min-width: 50%;
+    }
+  }
+`;
+
+export const UsefulLinksLi = ({ children, ...props }) => (
```

Comment:
Why do we need to have a rest operator here  `...props`?

Reply:
@sadick254 We should agree on this to be consistent across Psammead. 
There are components that follow this approach like the [Navigation](  <StyledUnorderedList role="list" {...props}>
), to add `attributes` and in case we want to pass more props when invoking them. 
But others, like the [StoryPromo](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-story-promo-list/src/index.jsx#L46) adds the attributes in the styled component declaration.

Reply:
None that I am aware of, so we should go for the second approach, but still, we should keep it in mind to be always consistent.

Reply:
@lokh01 Well in this case since there is `...props` we could pass down literally any props in case we wanted/needed to. Right now we are not passing anything though and I am going to rewrite the way `UsefulLinksLi` and `UsefulLinksUl` are written so that they don't take in any props.